be/src/storage/rowset/rowset_writer.cpp

```diff
@@ -404,8 +408,13 @@ Status HorizontalRowsetWriter::flush_chunk(const Chunk& chunk, SegmentPB* seg_in
         break;
     case FlushChunkState::UPSERT:
         break;
+    case FlushChunkState::DELETE:
+        _flush_chunk_state = FlushChunkState::MIXED;
+        break;
+    case FlushChunkState::MIXED:
+        break;
     default:
-        return Status::Cancelled(_dump_mixed_segment_delfile_not_supported());
+        return Status::Cancelled("UNKNOWN flush chunk state");
```

Comment:
1. Add a tablet info in this error Status.

Reply:
> 1. Add a tablet info in this error Status.

done, changed in https://github.com/StarRocks/starrocks/pull/17264/commits/8527758eaac88fd366e07ed4bd2cdf2b3f757dde